src/analyses/variable-sensitivity/abstract_object.cpp

```diff
@@ -420,22 +420,16 @@ Function: abstract_objectt::update_last_written_locations
    An abstract_object_pointer pointing to the cloned, updated object.
 
  Purpose: Creates a mutable clone of the current object, and updates
```

Comment:
It doesn't create a mutable clone in this case so probably comment should be updated.